plugins/db_manager.py

```diff
@@ -174,6 +175,62 @@ async def acl_show(ctx: Context, acl: str) -> None:
 
     await ctx.send(format("{!b:yaml}", yaml.dump(data)))
 
+@acl_command.command("find")
+@privileged
+async def acl_find(ctx: Context, type: str, id: int) -> None:
```

Comment:
You could have `type: Literal["role", "channel", "category", "user"]` and the library would check that (and provide a better help signature for the command).